Project.toml

```diff
@@ -11,6 +11,10 @@ QEDcore = "35dc0263-cb5f-4c33-a114-1d7f54ab753e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
+Distributions = "0.25"
+DocStringExtensions = "0.9"
+QEDbase = "0.2.2"
```

Comment:
Is it really necessary to pin the patch version?

Reply:
I wondered the same thing and looked at what QEDcore did. It probably is necessary here because there were new features in 0.2.2 which this relies on (not entirely sure though). But the version specifiers here go upwards anyways so it should be able to take anything above and including 0.2.2 and below (excluding) 0.3.0, at least if I understood the documentation correctly.

Reply:
Indeed, it should include everything above 0.2.2 and below 0.3. Therefore it should be fine.